### ISSUE TITLE
Editor tries to create a file when a test fails

### ISSUE THREAD

**Issue by ervamate** [connor4312/nodejs-testing#7]
The runner works fine when all tests pass, but when it fails, I get the following screen:

<img width="541" alt="test result" src="https://user-images.githubusercontent.com/25302288/234256716-5056803b-0b20-4042-ab1d-8efc0592f2f5.png">

I'm on VSCode `Version: 1.77.3` and I believe the version of the extension.

I thought it was related to me using devcontainers, however it happens while running on my machine.

When I press `Create File`, it returns the following error:

```
Unable to write file '/file:///Users/<full-path-to-file>' (Unknown (FileSystemError): Error: EROFS: read-only file system, mkdir '/file:')
```

I also noticed that the gutter does not show the arrow icon, but maybe that's unrelated

LMK if I can provide more info.

**Comment by connor4312** [comment]
Thanks for the issue. I'm not able to repro this. My first suspect would be some kind of path sensitivity issue. Can you confirm that you're opening the file as `Tokenizer.js` and not something like `tokenizer.js`, and likewise that all parts of the file path in your workspace have the same casing as this incorrect file URI?

**Comment by ervamate** [comment]
I tried to reduce to the minimum setup. I created a new empty profile and installed only the extension. When I click "run tests" or run it with the keyboard shortcut and the test fails, here's what I get:

<img width="1450" alt="index test js — node-testing — nodejs-testing 2023-05-27 21-08-22" src="https://github.com/connor4312/nodejs-testing/assets/25302288/7eb5a699-3971-466c-bc32-70e795c13db7">

When I click "create file", here's what it shows:

<img width="260" alt="Monosnap 2023-05-27 21-09-30" src="https://github.com/connor4312/nodejs-testing/assets/25302288/c7d34f66-a6fb-48eb-823b-9acf916984dc">

My current dir is:

```
$ pwd
/Users/felipe/Projects/cbh/node-testing
```

Hopefully this helps. This doesn't seem to happen on successful tests.
